.packit.yaml

```diff
@@ -12,6 +12,8 @@ packages:
   podman-centos:
     pkg_tool: centpkg
     specfile_path: rpm/podman.spec
+  podman-rhel:
+    specfile_path: rpm/podman.spec
```

Comment:
Do you not need a pkg_tool like it set for the others?

Reply:
We don't. The rhel builds are automatically triggered once the centos stream build is done so we don't have to worry about actually building the rhel package. We have the `podman-rhel` stuff only for isolating rhel jobs from the others, so PRs from authors without access will still trigger the fedora and centos tests.